test/MUXControls.Test/AxeTestHelper.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+using Common;
+
+#if USING_TAEF
+using WEX.TestExecution;
+using WEX.TestExecution.Markup;
+using WEX.Logging.Interop;
+#else
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+#endif
+using Axe.Windows.Automation;
+using System.Diagnostics;
+
+namespace MUXTestInfra.Shared.Infra
+{
+    public class AxeTestHelper
+    {
+        private static IScanner scanner = null;
+        public static IScanner AxeScanner
+        {
+            get
+            {
+                if (scanner == null)
+                {
+                    LoadScanner();
+                }
+                return scanner;
+            }
+        }
+
+        private static void LoadScanner()
+        {
+            var processes = Process.GetProcessesByName("MUXControlsTestApp");
```

Comment:
>MUXControlsTestApp [](start = 56, length = 18)

Does this work with the inner loop?

Reply:
Yes it does, the process name is the same for both.